ehr/resources/views/ehrAdmin.html

```diff
@@ -1,96 +1,44 @@
 <script type="text/javascript">
 
-Ext4.onReady(function (){
+</script>
 
-if (LABKEY.Security.currentUser.isAdmin){
-    var webpart = <%=webpartContext%>;
+<html>
+<div class="col-lg-3">
+<div class="lead">
+    <h4><b>Admin:</b></h4>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/study-manageStudy.view?'>Manage EHR Study</a></div>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/ehr-populateLookupData.view?'>Populate Reports and Lookups</a></div>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/core-modulePropertyAdmin.view?moduleName=EHR'>Set EHR Properties</a></div>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/ehr-ensureQcStates.view?'>Add QC States</a></div>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/ehr-ehrTemplates.view?'>EHR Extensible Columns</a></div>
+    <div><a class="labkey-text-link" href='<%=contextPath%>/ehr<%=containerPath%>/primeDataEntryCache.view?'>Prime/Refresh EHR Data Cache</a></div>
```

Comment:
Can we use the "new" URL syntax consistently here? For example:

```suggestion
    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/ehr-primeDataEntryCache.view?'>Prime/Refresh EHR Data Cache</a></div>
```

Reply:
Fixed all links using the old URL style